Makefile

```diff
@@ -53,6 +54,10 @@ clean: clean-objects
 
 clean-all: clean clean-slang
 
+install: build/slang.so
+	$(YOSYS_CONFIG) --exec mkdir -p $(DESTDIR)$(PLUGINDIR)
+	$(YOSYS_CONFIG) --exec cp $< $(DESTDIR)$(PLUGINDIR)
```

Comment:
We are invoking yosys-config both here and in the assignment to the PLUGINDIR variable, I think one of those is superfluous

Reply:
Welcome :-) I just simplified what is done at `install`. Let me know what you think or what your proposal is.